Tournament_Calculator.py

```diff
@@ -0,0 +1,323 @@
+        'Second':{"Arakni":(sheet_obj.cell(row = (55), column = 3).value),"Azalea":(sheet_obj.cell(row = (55), column = 4).value),"Betsy":(sheet_obj.cell(row = (55), column = 5).value),"Bravo":(sheet_obj.cell(row = (55), column = 6).value),"DIO":(sheet_obj.cell(row = (55), column = 7).value),"Dash":(sheet_obj.cell(row = (55), column = 8).value),"Dorinthea":(sheet_obj.cell(row = (55), column = 9).value),"Fai":(sheet_obj.cell(row = (55), column = 10).value),"Kano":(sheet_obj.cell(row = (55), column = 11).value),"Kassai":(sheet_obj.cell(row = (55), column = 12).value),"Katsu":(sheet_obj.cell(row = (55), column = 13).value),"Kayo":(sheet_obj.cell(row = (55), column = 14).value),"Levia":(sheet_obj.cell(row = (55), column = 15).value),"Maxx":(sheet_obj.cell(row = (55), column = 16).value),"Olympia":(sheet_obj.cell(row = (55), column = 17).value),"Prism":(sheet_obj.cell(row = (55), column = 18).value),"Rhinar":(sheet_obj.cell(row = (55), column = 19).value),"Riptide":(sheet_obj.cell(row = (55), column = 20).value),"Boltyn":(sheet_obj.cell(row = (55), column = 21).value),"Teklovossen":(sheet_obj.cell(row = (55), column = 22).value),"Uzuri":(sheet_obj.cell(row = (55), column = 23).value),"Victor":(sheet_obj.cell(row = (55), column = 24).value),"Viserai":(sheet_obj.cell(row = (55), column = 25).value),"Vynnset":(sheet_obj.cell(row = (55), column = 26).value),"Enigma":(sheet_obj.cell(row = (55), column = 27).value),"Nuu":(sheet_obj.cell(row = (55), column = 28).value),"Zen":(sheet_obj.cell(row = (55), column = 29).value)}}}
+
+
+
+
+#Swiss Pairings Logic
+def generate_pairings(players, previous_pairings=None, results=None):
+    pairings = []
+    if previous_pairings is None:
+        previous_pairings = set()  # Initialize an empty set for previous pairings
+    if results is None:
+        results = {}  # Initialize an empty dictionary for results
+    
+    # Group players based on their current results
+    result_groups = {}

```

Comment:
Another readability thing that would also help with your skill development / making code look professional would be to use type hints everywhere (https://dagster.io/blog/python-type-hinting). Especially with things like dicts, seeing up front what the key and value types are can be helpful in understanding how it's going to be used. In addition, it sets you up for success if you switch to a language that has stronger typing rules and also helps drill some best practices into you around making sure you're confident on what type will be in each variable.